Description: Fixes ILL_ILLOPN crash in Yakkety upon startup because of webkit
not being updated for Linux 4.5.

Index: beta/third_party/WebKit/Source/wtf/allocator/PageAllocator.cpp
===================================================================
--- beta.orig/third_party/WebKit/Source/wtf/allocator/PageAllocator.cpp
+++ beta/third_party/WebKit/Source/wtf/allocator/PageAllocator.cpp
@@ -41,6 +41,9 @@
 #include <errno.h>
 #include <sys/mman.h>
 
+// Added in Linux 4.5, but we don't want to depend on 4.5 at runtime
+#undef MADV_FREE
+
 #ifndef MADV_FREE
 #define MADV_FREE MADV_DONTNEED
 #endif
